mathics/core/element.py

```diff
@@ -422,12 +422,28 @@ def user_hash(self, update) -> None:
         # __hash__ might only hash a sample of the data available.
         raise NotImplementedError
 
-    def to_sympy(self, **kwargs):
+    def to_python(self, *args, python_form: bool = False, **kwargs):
+        # Returns a native builtin Python object
+        # something in (int, float, complex, str, tuple, list or dict.).
+        # (See discussions in
+        #  https://github.com/Mathics3/mathics-core/discussions/550
+        # and
+        # https://github.com/Mathics3/mathics-core/pull/551
+        #
+        #
+        # if n_evaluation is an Evaluation object, then the expression
+        # is passed by an eval_N().
+        # If python_form is True, the standard behaviour is changed,
```

Comment:
Please let's find and remove any use of to_python() with `python_form=True` and remove this parameter.

Reply:
Sure!